specification/hybriddatamanager/resource-manager/Microsoft.HybridData/stable/2016-06-01/hybriddata.json

```diff
@@ -0,0 +1,2608 @@
+          "description": "Name of the object.",
+          "type": "string",
+          "readOnly": true
+        },
+        "id": {
+          "description": "Id of the object.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "Type of the object.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "Error": {
+      "description": "Top level error for the job.",
+      "required": [
+        "code"
+      ],
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Error code that can be used to programmatically identify the error.",
+          "type": "string"
+        },
+        "message": {
+          "description": "Describes the error in detail and provides debugging information.",
+          "type": "string"
+        }
+      }
+    },
+    "ErrorDetails": {
+      "description": "Error Details",
+      "type": "object",
+      "properties": {
+        "errorMessage": {
+          "description": "Error message.",
+          "type": "string"
+        },
+        "errorCode": {
+          "format": "int32",
+          "description": "Error code.",
+          "type": "integer"
+        },
+        "recommendedAction": {
+          "description": "Recommended action for the error.",
+          "type": "string"
+        },
+        "exceptionMessage": {
+          "description": "Contains the non localized exception message",
+          "type": "string"
+        }
+      }
+    },
+    "Job": {
+      "description": "Data service job.",
+      "required": [
+        "status",
+        "startTime",
+        "properties"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/DmsBaseObject"
+        }
+      ],
+      "properties": {
+        "status": {
+          "description": "Status of the job.",
+          "enum": [
+            "None",
+            "InProgress",
+            "Succeeded",
+            "WaitingForAction",
+            "Failed",
+            "Cancelled",
+            "Cancelling"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "JobStatus",
+            "modelAsString": false
+          }
+        },
+        "startTime": {
+          "format": "date-time",
+          "description": "Time at which the job was started in UTC ISO 8601 format.",
+          "type": "string"
+        },
+        "endTime": {
+          "format": "date-time",
+          "description": "Time at which the job ended in UTC ISO 8601 format.",
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/JobProperties",
+          "description": "Job properties.",
+          "x-ms-client-flatten": true
+        },
+        "error": {
+          "$ref": "#/definitions/Error",
+          "description": "Top level error for the job."
+        }
+      }
+    },
+    "JobDefinition": {
```

Comment:
The 200 response model for an ARM PUT operation must have x-ms-azure-resource extension set to true in its hierarchy

Reply:
Waiting for a suppression on this issue.

Reply:
Suppression added